webapp/portlet/src/main/webapp/skin/less/common/PeopleList/Style-mobile.less

```diff
@@ -5,6 +5,9 @@
   .inputPeopleFilter {
     max-width: none;
   }
+  .selectPeopleFilter {
+    width: 100px;
```

Comment:
Why `100px` and why adding anew file for tablet ? please reduce the CSS by merging both tablet & mobile styles.
Will we apply this same UX in other filters in other pages, if yes, we should add a CSS helper to define the filter field style for all applications. By example, instead of using `width: 100px` we can use `max-width: 100px` in new CSS helper and let the container decide which width to add for the input.